frontend/app/views/event/whatson.scala.html

```diff
@@ -0,0 +1,118 @@
+@(pageInfo: model.PageInfo, eventPortfolio: model.EventCollections, membershipFrontArticles: Seq[model.MembersOnlyContent])
+
+@main("Events", pageInfo=pageInfo) {
+    <main role="main" class="l-constrained">
+
+        <section class="page-slice l-inset">
+            <h3 class="h-section h-section--lead">Trending</h3>
+            <ul class="grid grid--4up grid--bordered grid--single-row">
+            @for(event <- eventPortfolio.trending.take(4)) {
+                <li class="grid__item">
+                @fragments.event.item(event)
+                </li>
+            }
+            </ul>
+        </section>
+
+        <section class="page-slice l-inset">
+            <h3 class="h-section h-section--lead">This week</h3>
+            <ul class="grid grid--4up grid--bordered grid--single-row">
+                @for(event <- eventPortfolio.thisWeek.take(4)) {
+                    <li class="grid__item">
+                        @fragments.event.item(event)
+                    </li>
+                }
+            </ul>
+        </section>
+
+        <section class="page-slice l-inset">
+            <h3 class="h-section h-section--lead">Next Week</h3>
+            <ul class="grid grid--4up grid--bordered grid--single-row">
+            @for(event <- eventPortfolio.nextWeek.take(4)) {
+                <li class="grid__item">
+                @fragments.event.item(event)
+                </li>
+            }
+            </ul>
+        </section>
+
+        <section class="page-slice l-inset">
+            <h3 class="h-section h-section--lead">What's Hot</h3>
+            <ul class="grid grid--4up grid--bordered grid--single-row">
+                @for(event <- eventPortfolio.topSelling.take(4)) {
+                    <li class="grid__item">
+                        @fragments.event.item(event)
+                    </li>
+                }
+            </ul>
+        </section>
+
+        <section class="page-slice l-inset">
```

Comment:
I know this is a spike but would it be worth making this a reusable fragment that takes a title, a `Sequence` and a count (4)?